src/wagtailtrans/models.py

```diff
@@ -101,6 +102,10 @@ class TranslatablePage(Page):
 
     is_creatable = False
 
+    search_fields = Page.search_fields + [
+        FilterField('language_id'),
```

Comment:
Indexing the language seems like a good idea, couldn't we add the `language__code` instead of the id?